gammapy/maps/axes.py

```diff
@@ -1714,7 +1730,11 @@ def from_table(cls, table, format="gadf"):
                 if axcols is None:
                     break
 
-                axis = MapAxis.from_table(table, format=format, idx=idx)
+                # TODO: what is good way to check whether it is a time axis?
+                try:
```

Comment:
Should we add tags for axes too?
I guess that this will break the gadf format.

Reply:
Yes, this seems like a good long-term solution. However I think for now we can stick with the `try` and `error` block. I think it's safe: if there is a reference time present in the axis specification then it will create a `TimeMapAxis` otherwise a `MapAxis`.